### ISSUE TITLE
Visualize training

### ISSUE THREAD

**Issue by ahmedreyad2019** [chensong1995/HybridPose#45]
Is there a way to visualize the training curve?

**Comment by chensong1995** [comment]
Hi ahmedreyad2019,

The training script prints out the training loss to stdout. You can add a few lines in the training script to create a Tensorboard chart.
